source/tyr/migrations/versions/962cfcce76f_block_until_column_added_for_.py

```diff
@@ -0,0 +1,21 @@
+"""'block_until' column added for restriction access before this date.
+
+Revision ID: 962cfcce76f
+Revises: 11eae2b0e6b0
+Create Date: 2015-12-16 11:46:05.960382
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '962cfcce76f'
+down_revision = '11eae2b0e6b0'
+
+from alembic import op
+import sqlalchemy as sa
+
+def upgrade():
+    op.add_column('user', sa.Column('block_until', sa.DateTime(), nullable=True))
```

Comment:
Choose:
- In application, convert the time to UTC and stored using a TIMESTAMP WITH TIME ZONE column.
In this case, add "timezone=True"
- Or set database's timezone to UTC.